radix-engine/src/blueprints/consensus_manager/events/consensus_manager.rs

```diff
@@ -10,6 +10,6 @@ pub struct RoundChangeEvent {
 pub struct EpochChangeEvent {
     /// The *new* epoch's number.
     pub epoch: u64,
-    /// The *new* epoch's validator set.
-    pub validators: BTreeMap<ComponentAddress, Validator>,
+    /// The *new* epoch's validator set, ordered by stake descending.
+    pub validators: IndexMap<ComponentAddress, Validator>,
```

Comment:
instead of repeated `rustdoc + IndexMap`, could we have a single `ActiveValidatorSet` struct, re-used in the `EpochChangeEvent` and `CurrentValidatorSetSubstate`?

(it would also be a _perfect_ place for other repeated code like `.get_by_index(ValidatorIndex)` and `.get_by_address(ComponentAddress)` and `.total_active_stake()` etc)